services/dashboard/views/cluster/workers/workers.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+
+angular.module('dashboard.cluster')
+
+  .config(['$routeProvider',
+    function ($routeProvider) {
+      $routeProvider
+        .when('/cluster/workers', {
+          label: 'Workers',
+          templateUrl: 'views/cluster/workers/workers.html',
+          controller: 'WorkersCtrl'
+        });
+    }])
+
+  .controller('WorkersCtrl', ['$scope', 'restapi', function ($scope, restapi) {
+    $scope.slotsUsageClass = function (usage, good, concern, bad) {
+      return usage < 50 ? good : (usage < 75 ? concern : bad);
+    };
+    $scope.statusClass = function (status, good, bad) {
+      return status === 'active' ? good : bad;
+    };
+
+    restapi.subscribe('/workers', $scope,
```

Comment:
Add reference here to the data type WorkerData. Something like:
//TODO datatype: WorkerData. Try and do serde using scalajs See #458